.gitignore

```diff
@@ -457,3 +457,7 @@ exepublish/
 
 # ignore cucumber/aruba temporary files
 tmp/
+
+# ignore Ruby bundle/vendor files
+.bundle/
+vendor/
```

Comment:
These dictionaries are created on Linux when not providing a `sudo` password, as many Linux users are cautious about installing system-wide packages by a means other than the system package manager; in this case, the `bundle install` command recommends (and provides command for) installing local to the directory, producing these.

On that note, `.bundle/config` can also be rather system-specific, and thus not portable.

I assume what you're saying here is that this is fine for now, but we keep an eye on it, right?

Reply:
You understood me correctly. It's fine to leave these files ignored for now, but I'm trying to make a mental note in case we run into Ruby weirdness in the future. :smile: